### DISCUSSION TITLE
stable scores differ on the leaderboard and also in the replay

### DISCUSSION THREAD

**Discussion by BlauFx** [ppy/osu#17625] [Q&A] [answered]
Hello there. My question is why don't stable scores in lazer (classic mod applied) equal to the score shown on the stable leaderboard?
If I watch a replay in lazer, the score of the play is also neither equal to the shown score on lazer or the one on stable. 

https://user-images.githubusercontent.com/34138240/161421173-cabadc8c-34fa-47b9-b122-b6725cf4050f.mp4

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/11311
https://github.com/ppy/osu/discussions/17328

**Reply by BlauFx** [reply]
This kinda of explains why the replay score may differ from the leaderboard score. However just looking at the leaderboard shouldn't lazer show the exact same leaderboard as stable? Nothing is being recalculated (I think). I'm assuming lazer is pulling those scores from the database where all stable scores are being stored so those scores should be equal to the one on the stable leaderboard?

**Reply by bdach** [reply]
scores in the leaderboard are currently being recalculated to match lazer's scoring styles, based on the hit statistics returned by web.
